vue.config.js

```diff
@@ -16,6 +16,8 @@ process.env.VUE_APP_BENEFIT_COUNT = count
 process.env.VUE_APP_MATOMO_ID = matomo.id
 process.env.VUE_APP_VALIDATION_DELAY = (animation && animation.delay) || 0
 process.env.VUE_APP_BASE_URL = baseURL
+process.env.VUE_APP_DEMO = process.env.CONTEXT === "deploy-preview"
```

Comment:
```suggestion
process.env.VUE_APP_CONTEXT = process.env.CONTEXT === "deploy-preview" ? "demo" : "production"
```
:thinking: 

Comme les propriétés de `process.env` sont forcément des chaînes de caractères je me demande quelle meilleure alternative à "true" pourrait être choisie.

Reply:
Dans ce cas pourquoi ne pas stocker directement
process.env.VUE_APP_CONTEXT = process.env.CONTEXT

et faire `process.env.VUE_APP_CONTEXT === "deploy-preview"`

Reply:
Oui carrément